src/reducers/polls.js

```diff
@@ -0,0 +1,217 @@
+import { createReducer } from '../utils/redux';
+
+// Mock Poll Data ----------------------------------------------
+
+const pollId1 = '1';
+const pollId2 = '2';
+
+const mockParsedAllPollsData = [
+  {
+    creator: '0xeda95d1bdb60f901986f43459151b6d1c734b8a2',
+    pollId: pollId1,
+    voteId: 'QmaozNR7DZHQK1ZcU9p7QdrshMvXqWK6gpu5rmrkPdT3L4',
```

Comment:
The poll service uses the `pollId` whereas the new CMS uses `voteId`, not a huge deal, but I think we should standardize this.

Reply:
👍good call. do you prefer one or the other?

Reply:
It seems the benefit of using voteIds is that the CMS doesn't need to know about pollIds at all.  However, its possible for two polls to have the same voteId.  That shouldn't happen since we're only getting polls created by whitelisted accounts and the risk team could make sure not to create the same poll twice, but other developers may want to use the poll service without the assumption of a whitelist, so using poll id might be necessary.

Reply:
Right, just to make sure I've got it straight -- `voteId` is the same as `multiHash` in this case? Is this what you felt it should be in the CMS too @b-pmcg?

Reply:
Yep, my understanding was that the `multihash` would be the `voteId` in the CMS.

Reply:
@tyler17 I see what you mean now. The `voteId` in the CMS doesn't necessarily need to be the multihash. It could be the generated `pollId`. One drawback I see with that involves the workflow of an admin user creating a poll document in the gov dashboard UI, and then creating a new poll in the CMS. How would they have access to the `pollId` from the contract? I see that it is emitted with the `PollCreated` event, but is that data immediately returned to the user?

Keeping both ID fields would bridge that gap, but I'm not sure if its good practice to maintain 2 ID fields.

cc @jparklev